boilerplate/registry/index.html

```diff
@@ -0,0 +1,237 @@
+        details of the assumptions made to generate this document.</p>
+    </section>
+    <section id="sotd">
+      <p>This document is intended to be the basis of other Registry documents
+        published by TTWG and is not intended to be published as a Registry itself.
+        The following SOTD text is auto-generated by the Respec tool to reflect
+        what an actual Registry Definition document would contain.</p>
+    </section>
+    <section class="informative">
+      <h2>Introduction</h2>
+      <p>Some informative introductory text about this Registry.</p>
+    </section>
+    <section id="registry-definition">
+      <h2>Registry Definition</h2>
+      <p class="ednote">Insert sections on custodianship and change process.</p>
+      <section id="custodianship">
+        <h3>Custodianship</h3>
+        <p>The <dfn data-cite="w3c-process#custodian">custodian</dfn> of this Registry is the <a>TTWG</a>. 
+          If the <a>TTWG</a> is unable to fulfil the role of <a>custodian</a>,
+          for example if it has been <a data-cite="w3c-process#GeneralTermination">closed</a>,
+          the <a>custodian</a> in lieu is the <a data-cite="w3c-process#Team">W3C Team</a>.
+        </p>
+        <p class="issue" data-number="244"></p>
+      </section>
+      <section id="change-process">
+        <h3>Change Process</h3>
+        <section id="requesting-a-change">
+          <h4>Requesting a change</h4>
+          <p>Changes to this [=W3C Registry=] MUST be requested (the <dfn>change request</dfn>) using any one of the following options:</p>
+          <ul>
+            <li>Open an issue on the Registry Table's version control system;</li>
+            <li>Send an email to the <a>TTWG</a>'s public email reflector
+              <a href="mailto:public-tt@w3.org">public-tt@w3.org</a></li>
+          </ul>
+          <p>The <a>change request</a> MUST include enough information for the <a>custodian</a>
+            to be able to identify all of:</p>
+          <ul>
+            <li>The Registry Table;</li>
+            <li>The Entries being proposed for addition;</li>
+            <li>The Provisional Entries for which changes are being proposed;</li>
+            <li>The Entries being proposed for a Status change;</li>
+            <li>Details of the requested changes for each Entry;</li>
+            <li>The motivation for making the requested changes;</li>
+            <li>Any other supporting information, for example real world usage statistics;</li>
+          </ul>
+          <p>The proposer of the change MAY open a pull request (or equivalent) on
+            the Registry Table's version control system, with that pull request
+            containing the proposed changes. If a pull request is opened then
+            a corresponding issue MUST also be opened and
+            the pull request MUST be linked to that issue.</p>
+          <aside class="note">A pull request representing the proposals made in the <a>change request</a>
```

Comment:
Is it better to move this note into next section, where an assessment is paused until a PR is opened by a TTWG member?

Reply:
I put it here deliberately so that anyone requesting a change can better understand the reason why they might want to open a pull request. The note text mentions TTWG review specifically - I could tweak it so that it says that TTWG review is needed when TTWG is the custodian, but it seemed too complicated. What do you think?

Reply:
That's a good reason... hrm,, how about to add some text about "if custodian is TTWG" here?
Of course, TTWG review will not start if custodian is not TTWG, and I understand such text could be verbose and duplicate.

Reply:
That was my thinking for writing it the way I did.

Reply:
thank you for clarification!